UI/window-basic-auto-config.cpp

```diff
@@ -878,6 +878,10 @@ inline const char *AutoConfig::GetEncoderId(Encoder enc)
 		return SIMPLE_ENCODER_QSV;
 	case Encoder::AMD:
 		return SIMPLE_ENCODER_AMD;
+#ifdef ENABLE_HEVC
+	case Encoder::NVENC_HEVC:
+		return SIMPLE_ENCODER_NVENC_HEVC;
+#endif
```

Comment:
Let's not add encoders to auto-config unless we modify the selection flow.

Reply:
Done.